karma.conf.js

```diff
@@ -55,7 +55,7 @@ module.exports = (config) => {
     reporters: [ 'mocha' ],
 
     files: [
-      'node_modules/babel-polyfill/dist/polyfill.js',
+      'node_modules/es6-shim/es6-shim.js',
```

Comment:
fwiw since this is JS, this can probably do `path.relative(require.resolve('es6-shim'), process.cwd())` or something similar